vcd/resource_vcd_org_vdc.go

```diff
@@ -414,11 +414,13 @@ func getVcdVdcInput(d *schema.ResourceData, vcdClient *VCDClient) (*types.VdcCon
 	}
 
 	if resourceGuaranteedMemory, ok := d.GetOk("memory_guaranteed"); ok {
-		params.ResourceGuaranteedMemory = resourceGuaranteedMemory.(float64)
+		value := resourceGuaranteedMemory.(float64)
```

Comment:
I know this is not how it should be in the end, but having a breaking patch merged into govcd forces me to fix it.
It will have to be aligned with https://github.com/terraform-providers/terraform-provider-vcd/pull/275